tests/subsurfaces.cpp

```diff
@@ -470,6 +470,40 @@ TEST_P(SubsurfaceTest, one_subsurface_to_another_fallthrough)
                     wl_fixed_from_int(pointer_y_2 - subsurface_top_y))));
 }
 
+TEST_P(SubsurfaceTest, place_below_simple)
+{
+    Test::RecordProperty("wlcs-skip-test", "Mir does not yet support subsurface reordering");
```

Comment:
We should not be implementing Mir specific limitations in these tests.

Any tests that Mir expects to fail ought to be "xfail"ed by the Mir integration code.

Reply:
Fixed

Reply:
You've removed the `FAIL()`, but you should also remove the mark-as-skipable.